src/private/types.php

```diff
@@ -0,0 +1,18 @@
+<?hh
```

Comment:
If this is v4.0, we can make the breaking change of removing this file again and making the unknown symbol errors `keyset<T>`s.

Reply:
The public API is still HackCollection based.
Since this is going to be a breaking change, would we want to go Hack arrays in 4.0.0?

Reply:
Nice catch :)

Reply:
:fishing_pole_and_fish: